src/main/java/org/opensearch/ad/transport/SearchAnomalyResultTransportAction.java

```diff
@@ -90,19 +90,21 @@ protected void doExecute(Task task, SearchRequest request, ActionListener<Search
 
         String[] concreteIndices = indexNameExpressionResolver
             .concreteIndexNames(clusterService.state(), IndicesOptions.lenientExpandOpen(), indices);
-        if (concreteIndices == null || concreteIndices.length == 0) {
-            // No result indices found, will throw exception
-            listener.onFailure(new IllegalArgumentException("No indices found"));
-            return;
-        }
+        // If concreteIndices is null or empty, don't throw exception. Detector list page will search both
+        // default and custom result indices to get anomaly of last 24 hours. If throw exception, detector
+        // list page will throw error and won't show any detector.
```

Comment:
Can you add a sentence about why this case is ok for the scenario of a cluster with no custom result indices + some new non-custom-result-detector that hasn't finished one interval (where no default result index exists)?

Reply:
Sure